code.json

```diff
@@ -42,6 +42,52 @@
       "email": "jlaura@usgs.gov"
     },
 
+    "date": {
+      "metadataLastUpdated": "2023-04-07"
+    }
+  }, {
+    "name": "ale",
+    "organization": "U.S. Geological Survey",
+    "description": "GitLab repository for the Abstraction Layer for Ephemerides package",
+    "version": "0.8.8",
```

Comment:
0.9.0? Based on the tagged releases, it looks like this should have a 0.8.8 and a 0.9.0 entry?